packages/asset-swapper/src/constants.ts

```diff
@@ -43,6 +43,8 @@ const DEFAULT_SWAP_QUOTER_OPTS: SwapQuoterOpts = {
     },
     ...DEFAULT_ORDER_PRUNER_OPTS,
     samplerGasLimit: 250e6,
+    rfqtTakerApiKeyWhitelist: [],
```

Comment:
consider pushing these down into an `rfqt` namespace.
```javascript
rfqt: {
  apiKeys: [],
  endpoints: [],
}
```

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/93872ad7a30cd98bca8c4fe14087cb5b732b42b9